pyjobs/core/tests/test_job_views.py

```diff
@@ -117,3 +117,24 @@ def test_check_applied_for_job(self):
         request = request_client.content.decode('utf-8')
         expected_response = "Aplicar para esta vaga pelo PyJobs"
         self.assertTrue(expected_response in request)
+
+    def test_check_if_profile_with_no_skills_can_apply(self):
+        self.client.login(username="jacob", password="top_secret")
+        job_url = "/job/{}/".format(self.job.pk)
+        request_client = self.client.get(job_url)
+        request = request_client.content.decode('utf-8')
```

Comment:
This is not a `request`. This is the contents of a response converted from bytes to string.